routes/web.php

```diff
@@ -46,6 +46,8 @@
         Route::post('/dashboard/setTab', 'DashboardController@setTab');
     });
 
+    Route::get('/store/{file}', 'StorageController@download')->where('file', '.*')->name('download');
```

Comment:
It's weird to have a `GET` route that is actually called `store` and the method in the controller is called `download`.

Reply:
`storage` is not usable. What would you use? `files`?
The function can be called `get` indeed.